[#686] Transform all JMP to category.json

ifirmawan opened the pull request:
## TODO/DONE

Transform all JMP to category.json

- [x] Household
- [x] Wash in schools
- [x] Urban sanitation

* Note:  Health facilities has been transformed as initial category.json

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204676393419434

dedenbangkit reviewed (changes requested):
@ifirmawan please re-check the structure for overall categories, some of them are miss-match. e.g:
- "No Service" become `false` in Health Facilities for Sanitation category.
or; perhaps seeder doesn't comply the dependency structure.

dedenbangkit reviewed (changes requested):
@ifirmawan please re-check the structure for overall categories, some of them are miss-match. e.g:
- "No Service" become `false` in Health Facilities for Sanitation category.
or; perhaps seeder doesn't comply the dependency structure.

ifirmawan commented:
> @ifirmawan please re-check the structure for overall categories, some of them are miss-match. e.g:
> 
> * "No Service" become `false` in Health Facilities for Sanitation category.
>   or; perhaps seeder doesn't comply the dependency structure.

Hi mas @dedenbangkit, I can confirm this issue from the category.json, I've fixed it and added more tests
